ibm/mas_devops/playbooks/mas/configure-suite-db.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: localhost
+  any_errors_fatal: true
+  vars:
```

Comment:
All these vars can be removed from the playbook as these can be set using environment variables. See other playbooks that are the bare minimum.

Reply:
Added the variables under the default folders